src/stan/lang/grammars/statement_grammar_def.hpp

```diff
@@ -90,131 +90,131 @@ namespace stan {
       using boost::spirit::qi::labels::_b;
       using boost::spirit::qi::labels::_r1;
       using boost::spirit::qi::labels::_r2;
-      using boost::spirit::qi::labels::_r3;
-      using boost::spirit::qi::labels::_r4;
 
       using boost::phoenix::begin;
       using boost::phoenix::end;
 
-      // inherited features
-      //   _r1 true if sample_r allowed
-      //   _r2 source of variables allowed for assignments
-      //   _r3 true if return_r allowed
-      //   _r4 true if in loop (allowing break/continue)
+      // inherited features _r1, _r2
```

Comment:
I don't think you even need to say "inherited features" --- you should assume the reader of the doc knows the API.